modules/core/03-connection/keeper/handshake_test.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/cosmos/ibc-go/v5/modules/core/03-connection/types"
 	host "github.com/cosmos/ibc-go/v5/modules/core/24-host"
 	"github.com/cosmos/ibc-go/v5/modules/core/exported"
-	ibctmtypes "github.com/cosmos/ibc-go/v5/modules/light-clients/07-tendermint/types"
+	ibctmtypes "github.com/cosmos/ibc-go/v5/modules/light-clients/07-tendermint"
```

Comment:
I'm curious what we should do with this alias. We currently use `tendermint` in most places, but maybe we should use `ibctm`?